test/test_integration_cluster.rb

```diff
@@ -133,6 +133,32 @@ def test_stuck_phased_restart
     worker_respawn { |phase0_worker_pids| Process.kill :USR1, @pid }
   end
 
+  # It does not wipe out BUNDLE_GEMFILE et al
+  def test_usr2_restart_preserves_bundler_environment
+    skip_unless_signal_exist? :USR2
```

Comment:
I don't think this skip is necessary. `preload_bundler` can be triggered by starting any Puma cluster that doesnt `preload_app!`.

Reply:
Then again, I'm not sure how you test it without a USR2 restart, so maybe ignore me?